packages/examples/packages/interactive-ui/src/index.test.tsx

```diff
@@ -43,6 +43,8 @@ describe('onRpcRequest', () => {
 
       await formScreen.selectInDropdown('example-dropdown', 'option3');
 
+      await formScreen.chooseFromRadioGroup('example-radiogroup', 'option3');
```

Comment:
Could we be uniform with Dropdown and call it `selectFromRadioGroup` ?

Reply:
Updated.